content/events/filecoin-foundation-hacker-base-ethcc.md

```diff
@@ -1,7 +1,10 @@
 ---
-eventName: "Filecoin Hacker Base at EthCC"
-website: 
+eventName: "Filecoin Network Base at EthCC"
```

Comment:
@laeccles I just updated this event as per Ryo's inputs (including the event name) - Can you please check with Ryo on this - as the final content to be updated.

Reply:
@WinsCodes I have confirmed the name change with Ryo and corrected the dates.  Please update this in the events page.